src/future-utils.js

```diff
@@ -0,0 +1,47 @@
+const SCORES = {
+  touchdown: 7,
+  fieldgoal: 3,
+  touchdownExtra: 8,
+  touchdownMiss: 6,
+  safety: 2,
```

Comment:
A PR comment seems like the wrong place for me to explain NFL rules, but there are two more possible scoring scenarios that only happen on the untimed down that follows a touchdown. 

1. The team that scored can kick an extra point to add 1pt to their score
2. The team that was scored upon can force a turnover and run the ball back for 2pts

These are corner cases. If we're polling only once every 30s it's unlikely we're  in an untimed down.